compiler/frontend/src/org/jetbrains/kotlin/effectsystem/adapters/ValueIDs.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jetbrains.kotlin.effectsystem.adapters
+
+import org.jetbrains.kotlin.effectsystem.structure.ConstantID
+import org.jetbrains.kotlin.effectsystem.structure.ESValueID
+import org.jetbrains.kotlin.resolve.calls.smartcasts.DataFlowValue
+
+class DataFlowValueID(val dfv: DataFlowValue) : ESValueID {
```

Comment:
Not quite sure this wrapper is needed. Can `DataFlowValue` itself serve you here?